yew-macro/src/html_tree/html_tag/tag_attributes.rs

```diff
@@ -161,6 +282,16 @@ impl Parse for TagAttributes {
 
         let mut listeners = Vec::new();
         for listener in TagAttributes::drain_listeners(&mut attributes) {
+            if !TagAttributes::is_listener_supported(&listener) {
```

Comment:
I think we could drop the `cfg_if` dependency and add a `#[cfg(feature = "std_web")]` above this line

Reply:
The idealist in me probably got the better of me here 😅. I wanted to make it flexible enough so that it could be used for the case where stdweb supports an event that web-sys doesn't but it seems like a bit of a stretch in hindsight.

Reply:
I went ahead and inlined the entire function since it's only one line now. Feel free to complain if you want me to keep it as `is_listener_supported`.

Reply:
Haha, no worries! Thanks for the consolidation :)